server/routes/referrals/referralFormPresenter.test.ts

```diff
@@ -4,85 +4,184 @@ import serviceCategoryFactory from '../../../testutils/factories/serviceCategory
+            status: ReferralFormStatus.CannotStartYet,
+            tasks: [{ title: 'Check your answers', url: 'check-answers' }],
+          },
+        ]
 
-      const expected = [
-        {
-          type: 'single',
-          title: 'Retrieve service user record',
-          number: '1',
-          status: ReferralFormStatus.Completed,
-          tasks: [
-            { title: 'Enter service user case identifier', url: null },
-            { title: 'Confirm service user details', url: 'service-user-details' },
-          ],
-        },
-        {
-          type: 'single',
-          title: 'Find and select service user interventions',
-          number: '2',
-          status: ReferralFormStatus.Completed,
-          tasks: [
-            { title: 'Find and select service user interventions', url: null },
-            { title: 'Confirm interventions', url: null },
-          ],
-        },
-        {
-          type: 'single',
-          title: 'Review service user’s information',
-          number: '3',
-          status: ReferralFormStatus.Completed,
-          tasks: [
-            { title: 'Service user’s risk information', url: 'risk-information' },
-            { title: 'Service user’s needs and requirements', url: 'needs-and-requirements' },
-          ],
-        },
-        {
-          type: 'multi',
-          title: 'Add intervention referrals detail',
-          number: '4',
-          taskListSections: [
-            {
-              title: 'Accommodation referral',
-              number: '4.1',
-              status: ReferralFormStatus.InProgress,
-              tasks: [
-                { title: 'Select the relevant sentence for the accommodation referral', url: null },
-                { title: 'Select desired outcomes', url: 'desired-outcomes' },
-                { title: 'Select required complexity level', url: 'complexity-level' },
-                {
-                  title: 'What date does the accommodation service need to be completed by?',
-                  url: 'completion-deadline',
-                },
-                { title: 'Enter RAR days used', url: 'rar-days' },
-                { title: 'Further information for service provider', url: 'further-information' },
-              ],
-            },
-          ],
-        },
-        {
-          type: 'single',
-          title: 'Review responsible officer’s information',
-          number: '5',
-          status: ReferralFormStatus.NotStarted,
-          tasks: [{ title: 'Responsible officer information', url: null }],
-        },
-        {
-          type: 'single',
-          title: 'Check your answers',
-          number: '6',
-          status: ReferralFormStatus.CannotStartYet,
-          tasks: [{ title: 'Check your answers', url: 'check-answers' }],
-        },
-      ]
+        expect(presenter.sections).toEqual(expected)
+      })
+    })
+
+    describe('when the intervention details section has all required fields filled in', () => {
```

Comment:
Might this draft referral be a better example of the state described here if it _only_ has the required fields filled in, and the rest (e.g. `interpreterLanguage`) `null`?

Reply:
Good point. Have updated.

Reply:
By "optional", I mean the fields that are nullable on the `ReferralFields` type, so that excludes e.g. `additionalNeedsInformation`. Also, I think that `usingRarDays` should be `false` and `maximumRarDays` `null` here.

Reply:
Sorry - I understand what's going on now we've spoken - just `usingRarDays` needs changing I think.

Reply:
As discussed, we're leaving out the other "required" fields like `additionalNeedsInformation` as they live outside this intervention details section. Have updated `usingRarDays` though!